core/pkg/hostsensorutils/hostsensorgetfrompod.go

```diff
@@ -127,40 +127,40 @@ func (hsh *HostSensorHandler) GetVersion() (string, error) {
 // return list of LinuxKernelVariables
 func (hsh *HostSensorHandler) GetKernelVariables() ([]hostsensor.HostSensorDataEnvelope, error) {
 	// loop over pods and port-forward it to each of them
-	return hsh.sendAllPodsHTTPGETRequest("/LinuxKernelVariables", "LinuxKernelVariables")
+	return hsh.sendAllPodsHTTPGETRequest("/LinuxKernelVariables", LinuxKernelVariables)
```

Comment:
[Question]: Why do some of the paths begin with uppercase and some don't?

Reply:
It was like that for quite some time, but anyway, we have updated host-scanner to support case-insensitive for API requests.